lib/Backend/Sym.cpp

```diff
@@ -109,20 +112,24 @@ StackSym::New(IRType type, Func *func)
     return StackSym::New(func->m_symTable->NewID(), type, Js::Constants::NoRegister, func);
 }
 
+StackSym *
+StackSym::NewImplicitParamSym(Js::ArgSlot paramSlotNum, Func * func)
+{
+	return func->m_symTable->GetImplicitParam(paramSlotNum);
+}
 
 StackSym *
 StackSym::NewParamSlotSym(Js::ArgSlot paramSlotNum, Func * func)
 {
-    StackSym * stackSym = StackSym::New(func);
-    stackSym->m_paramSlotNum = paramSlotNum;
-    return stackSym;
+	return NewParamSlotSym(paramSlotNum, func, TyVar);
 }
 
 StackSym *
 StackSym::NewParamSlotSym(Js::ArgSlot paramSlotNum, Func * func, IRType type)
```

Comment:
> IRType type [](start = 64, length = 12)

nit: you can consider adding a default value for this parameter, and removing the other definition for NewParamSlotSym